tests/behaviour/typeql/steps.rs

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (C) 2022 Vaticle
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+use cucumber::{gherkin::Step, given, then, when};
+use futures::TryStreamExt;
+
+use crate::{behaviour::Context, generic_step_impl};
+
+generic_step_impl! {
+    #[step(expr = "typeql define")]
+    async fn typeql_define(world: &mut Context, step: &Step) {
+        // FIXME parse
+        world.transaction().query().define(step.docstring().unwrap()).await.unwrap();
```

Comment:
this is implementable right?

Reply:
and below

Reply:
@dmikhalin is already working on these fixmes in his fork.